cookbooks/aws-parallelcluster-slurm/recipes/config/config_head_node.rb

```diff
@@ -292,6 +292,7 @@
     region: node['cluster']['region'],
     munge_user: node['cluster']['munge']['user'],
     munge_group: node['cluster']['munge']['group'],
-    cluster_user: node['cluster']['cluster_user']
+    shared_directory: node['cluster']['shared_dir'],
```

Comment:
Nitpick: please rename this variable as `shared_directory_compute`.

Reply:
Done!